datasets/aardgasverbruik/aardgasverbruik_mra_liander/v1.0.0.json

```diff
@@ -0,0 +1,146 @@
+{
+  "id": "aardgasverbruik_mra_liander",
```

Comment:
Kan deze id in camelcase worden gedefinieerd dus (`aardgasverbruikMraLiander`). In postgres wordt het dan weer snakecase, maar in amsterdam schemas houden we de camelcase conventie aan.

Reply:
Liever ook geen domein-specifieke afkortingen in namen, omdat de schemas voor breed gebruik zijn bedoeld. Wel wordt de naam dan waarschijnlijk erg lang en is het verstandig een `shortname` te definieren die bepaalt hoe de tabelnaam in postgresql wordt. Zie: https://schemas.data.amsterdam.nl/docs/ams-schema-spec.html#shortname

Reply:
Aangepast naar AardgasverbruikMraLiander